source/hic_hal/nxp/lpc11u35/DAP_config.h

```diff
@@ -181,9 +187,21 @@ Disables the DAP Hardware I/O pins which configures:
 */
 static __inline void PORT_OFF(void)
 {
+#if !defined(PIN_nRESET_FET_DRIVE)
+    // open drain logic
     LPC_GPIO->DIR[PIN_nRESET_PORT] &= ~PIN_nRESET;
+#else
+    // FET drive logic
+    LPC_GPIO->DIR[PIN_nRESET_PORT] |= PIN_nRESET;
+    LPC_GPIO->CLR[PIN_nRESET_PORT] = PIN_nRESET;
+#endif
     LPC_GPIO->DIR[PIN_SWCLK_PORT] &= ~PIN_SWCLK;
     LPC_GPIO->DIR[PIN_SWDIO_PORT] &= ~PIN_SWDIO;
+
+#if defined(TARGET_POWER_HOLD)
+    //Release Target PowerHold
+    LPC_GPIO->CLR[PIN_PWH_PORT] = PIN_PWH;
+#endif
```

Comment:
Sorry for insufficient explanation.

<img src="https://docs.google.com/drawings/d/150pRk4Tabfq3527q06CBayEHYfAhrgQ_pG7auyApkhs/pub?w=953&amp;h=642">

Power hold of BlueNinja module the following two elements are responsible.

1. PowerSW (or PowerSW pin of Interface Board)
2. Power management by the software of TZ1000

If PowerSW is OFF, and the hold by the software is released, DC/DC converter must be the Disable.

Power hold by LPC11U35 of the interface is an auxiliary role to power ON only for the duration of the programming.